src/managed/OpenLiveWriter.Mshtml/MarkupPointer.cs

```diff
@@ -336,7 +336,10 @@ public IHTMLElement SeekElementRight(IHTMLElementFilter filter, MarkupPointer bo
         /// <param name="eAdj"></param>
         public void MoveAdjacentToElement(IHTMLElement element, _ELEMENT_ADJACENCY eAdj)
         {
-            PointerRaw.MoveAdjacentToElement(element, eAdj);
+            if (element != null)
```

Comment:
Sorry, but can you just revert the change here? I actually want it to crash :smile:. We are getting good crash data from this and there are a couple other instances we should go and fix too.